python/lsst/ap/association/ssoAssociation.py

```diff
@@ -171,18 +216,26 @@ def run(self, diaSourceCatalog, solarSystemObjects, exposure):
         self.metadata['nExpectedSsObjects'] = nSolarSystemObjects
         assocSourceMask = diaSourceCatalog["ssObjectId"] != 0
         unAssocObjectMask = np.logical_not(maskedObjects['associated'].values)
-        ssSourceData = pd.DataFrame(ssSourceData, columns=["ssObjectId", "obs_position_x", "obs_position_y",
-                                                           "obs_position_z", "obj_position_x",
-                                                           "obj_position_y", "obj_position_z"])
+        ssSourceData = pd.DataFrame(ssSourceData,
+                                    columns=[
+                                        "ssObjectId", "phaseAngle", "heliocentricDist", "topocentricDist"
+                                    ] + stateVectorColumns)
         ssSourceData["ra"] = ras
         ssSourceData["dec"] = decs
-        ssSourceData["expected_ra"] = expected_ras
-        ssSourceData["expected_dec"] = expected_decs
-        unassociatedObjects = maskedObjects[unAssocObjectMask].drop(columns=['obs_x_poly', 'obs_y_poly',
-                                                                             'obs_z_poly', 'obj_x_poly',
-                                                                             'obj_y_poly', 'obj_z_poly',
-                                                                             'obs_position', 'obj_position',
-                                                                             'associated'])
+        ssSourceData["residualRa"] = residual_ras
+        ssSourceData["residualDec"] = residual_decs
+        ssSourceData[source_column] = dia_ids
+        coords = SkyCoord(ra=ssSourceData['ra'].values * u.deg, dec=ssSourceData['dec'].values * u.deg)
```

Comment:
Are you certain this will always be given in degrees? It would be safer to retrieve the units in a similar way that you retrieve the values, if possible.